2312/1/koans/triangle.rb

```diff
@@ -0,0 +1,51 @@
+# Triangle Project Code.
+
+# Triangle analyzes the lengths of the sides of a triangle
+# (represented by a, b and c) and returns the type of triangle.
+#
+# It returns:
+#   :equilateral  if all sides are equal
+#   :isosceles    if exactly 2 sides are equal
+#   :scalene      if no sides are equal
+#
+# The tests for this method can be found in
+#   about_triangle_project.rb
+# and
+#   about_triangle_project_2.rb
+#
+# rubocop:disable Style/GuardClause
+# rubocop:disable Metrics/CyclomaticComplexity
+# rubocop:disable Metrics/PerceivedComplexity
+# :reek:FeatureEnvy
+# :reek:UtilityFunction
+def errors?(side_one, side_two, side_three)
```

Comment:
No need for all these `true` and `false`. The following method will do the same just replace conditions. And even better move them into separate methods
```
def errors?
  condition1 || condtion2
end
```